server/src/main/java/org/candlepin/model/CPRestrictions.java

```diff
@@ -24,21 +27,25 @@
 import java.util.Iterator;
 import java.util.List;
 
-
+import javax.inject.Inject;
 
 /**
  * The CPRestrictions class provides utility Criterion building methods to be used with Hibernate's
  * fluent-style query building.
  */
 public class CPRestrictions {
+
+    private Configuration config;
+
     private static class CPLikeExpression extends LikeExpression {
         public CPLikeExpression(String property, String value, Character escape, boolean ignoreCase) {
             super(property, value, escape, ignoreCase);
         }
     }
 
-    protected CPRestrictions() {
-        throw new UnsupportedOperationException("CPRestriction should not be instantiated");
+    @Inject
+    public CPRestrictions(Configuration config) {
+        this.config = config;
```

Comment:
I couldn't get it to work.  CPRestrictions now requires a reference to the Configuration object.  Guice does allow [static injection](https://github.com/google/guice/wiki/Injections#static-injections) but using it is not recommended and I thought it would making testing more difficult.

I will try again and see if I get it working.